### PR TITLE
Replaced --with-num-cores with --with-jobs in makejdk-any-platform.sh

### PR DESCRIPTION
**Pull request by neomatrix369**
Fixes issue https://github.com/AdoptOpenJDK/openjdk-build/issues/70

### CODE REVIEW

File: makejdk-any-platform.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ fi
 if [[ "$OS_MACHINE_NAME" == "armv7l" ]] ; then
    export JVM_VARIANT=${JVM_VARIANT:-zero}
    export MAKE_ARGS_FOR_ANY_PLATFORM=${MAKE_ARGS_FOR_ANY_PLATFORM:-"DEBUG_BINARIES=true images"}
-   export CONFIGURE_ARGS_FOR_ANY_PLATFORM=${CONFIGURE_ARGS_FOR_ANY_PLATFORM:-"--with-num-cores=4"}
+   export CONFIGURE_ARGS_FOR_ANY_PLATFORM=${CONFIGURE_ARGS_FOR_ANY_PLATFORM:-"--with-jobs=4"}
```
Comment (gdams) [inline makejdk-any-platform.sh]: please don't hard code the number (I suggest taking the value from `NUM_PROCESSORS` as this is already defined on all our build machines)

Comment (karianna) [inline makejdk-any-platform.sh]: Darn it, I missed that in my review.  I'll raise an issue.

Comment (neomatrix369) [inline makejdk-any-platform.sh]: \good catch @gdams , i'll fix it, in the next PR

Comment (neomatrix369) [inline makejdk-any-platform.sh]: Thinking of it again, its not necessary incorrect, as the `CONFIGURE_ARGS_FOR_ANY_PLATFORM` can be overriden with a new value which uses the NUM_PROCESSORS

Comment (neomatrix369) [inline makejdk-any-platform.sh]: with the new parameterising everything is dynamic and can be overriden, these values are not fixed

**Pull request merged**